pages/DonateStartPage.js

```diff
@@ -299,6 +300,9 @@ export default class DonateStartPage {
      * Choose email communication preferences.
      */
     async setCommsPreferences() {
+        // Add some grace pause to allow enough time for the checkboxes to render
```

Comment:
Can make this a bit more succinct - don't need to repeat the code in the comment.
```suggestion
        // Allow enough time for the checkboxes to render
```